pages/common/hut.md

```diff
@@ -0,0 +1,29 @@
+# hut
+
+> A CLI tool for sourcehut.
+> Use `hg` in place of `git` below if using mercurial.
+> See `man hut` for more details.
```

Comment:
```suggestion
> Use `hg` in place of `git` below if using mercurial. See `man hut` for more details.
> More information: <https://man.sr.ht>.
```

> [hut: A CLI tool for sr.ht](https://sr.ht/~emersion/hut/) is where you would link for that. However it says to read the man page to learn more. I don't mind linking it, but it seemed better to skip it in this case.

Agreed, seems like the man page website is WIP but we can still link it while moving the manpage suggestion to the second line (as we generally prefer having only 2 lines of page description and 1 line for more information link).